README.md

```diff
@@ -23,3 +23,9 @@ The specification uses WebIDL definitions, but TypeScript definititions are also
 ## Task Force Meeting Logistics
 
 See https://www.w3.org/WoT/IG/wiki/WG_WoT_Scripting_API_WebConf. 
+
+## Technical Matrix
+
+We use [labels](https://github.com/w3c/wot-scripting-api/issues) to categorize our work items.
```

Comment:
Isn't the wrong link? https://github.com/w3c/wot-scripting-api/labels

Reply:
Yes, I noticed that also :-) 
Thanks!